cmd/ent/ent_test.go

```diff
@@ -30,8 +33,29 @@ func TestCmd(t *testing.T) {
 	cmd = exec.Command("go", "run", "entgo.io/ent/cmd/ent", "generate", "./ent/schema")
 	stderr = bytes.NewBuffer(nil)
 	cmd.Stderr = stderr
-	require.NoError(t, cmd.Run(), stderr.String())
+	require.NoError(t, cmd.Run())
+	require.Zero(t, stderr.String())
 
 	_, err = os.Stat("ent/user.go")
 	require.NoError(t, err)
+
+	require.NoError(t, os.MkdirAll("migrations", 0750))
+	defer os.RemoveAll("migrations")
+
+	cmd = exec.Command(
+		"go", "run", "entgo.io/ent/cmd/ent",
+		"migrate", "diff",
+		"./ent/schema",
+		"--dir", "migrations",
+		"--dsn", "file:ent?mode=memory&_fk=1",
+		"--driver", "sqlite3",
```

Comment:
Use atlas/golang-migrate format? Where the driver/dialect is used as a prefix? 

```suggestion
		"--dsn", "sqlite://file:ent?mode=memory&_fk=1",
```

Reply:
I thought about that. But ent users are using the dsn without driver prefix everywhere and I wanted to stay consistent. 

If I make it clear in the docs though going with the prefixed version should be okay.

Reply:
Agree, but cli is different from programming one, and tbh, I wanted to change the `client.Open` API to "prefixed dsns" a long time ago, without breaking users' code.